fvp.mk

```diff
@@ -126,9 +174,16 @@ linux-cleaner: linux-cleaner-common
 # OP-TEE
 ################################################################################
 OPTEE_OS_COMMON_FLAGS += CFG_ARM_GICV3=y
-optee-os: optee-os-common
 
-optee-os-clean: optee-os-clean-common
+ifeq ($(USE_MEASURED_BOOT),1)
+	OPTEE_OS_COMMON_FLAGS += CFG_DT=y CFG_CORE_TPM_EVENT_LOG=y
+endif
+
+# Buildroot depends on optee-os, so make optee-os depend on the ftpm module in
+# order to have it ready before the former needs it.
```

Comment:
Not sure I get what you mean here, if it is what I think then why not just add:
```
buildroot: linux-ftpm-module
```
?

Reply:
I think my comment there is a bit misleading.

I needed to have `linux-ftpm-module` ready before buildroot starts building or otherwise the latter would fail as the module would not be ready when needed. So I considered adding `linux-ftpm-module` as a dependency of `buildroot`, as you suggested, however I was concerned that this could cause regressions on other platforms using `common.mk` as well, so instead, I decided that as `buildroot` already depends on `optee-os`, I could make the module depend on the latter, as this one is defined on `fvp.mk` and I know it will not cause regressions.

I will rephrase my comment to make it clear (unless you think there is a "cleaner" way to do it)

Reply:
You could still add `buildroot: linux-ftpm-module` to `fvp.mk`, no need to touch `common.mk`.

Reply:
What you may not realize is, a target can appear multiple times in a makefile as long as only one instance contains commands. It is convenient to add dependencies.

Reply:
Ahh, thanks for the suggestion, my Makefile skills are actually quite rusty.

Reply:
Rules fixed and smoke tests passed.